### PR TITLE
Why open source

### PR DESCRIPTION
**Pull request by benbalter**
Hoping to create a canonical URL for myself.

@bkeepers, @gbinal, @konklone obvs just an outline at this point, but if you have any other arguments for open source that you've heard / found persuasive, I'd love to hear them.

Fixes https://github.com/benbalter/benbalter.github.com/issues/313.

### CODE REVIEW

**Comment by benbalter** [comment]
@yourfrienderin far from a formed thought at this point, but once formed, a written, and much more articulate version of our conversation this afternoon.

File: _posts/2015-11-17-why-open-source.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+[Open Source (Almost) Everything](http://tom.preston-werner.com/2011/11/22/open-source-everything.html)
+
+* All the easy problems have already been solved
+* Shift developers from low-value work to high-value work
+* Modern software development practices (http://ben.balter.com/2015/09/21/open-source-behind-the-firewall/)
+* Great advertising
+* Force multiplier
+* Attract talent
+* Best technical interview possible
```
Comment (parkr) [inline _posts/2015-11-17-why-open-source.md]: (if willing to dig into process or follow dev for a while)

File: _posts/2015-11-17-why-open-source.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+### Macroeconomic
+
+* All the easy problems have already been solved
+* Stand on the shoulders of giants
+* Shift firms from low-value work to high-value work
+* Fancy name from @bkeepers blog post about standing on the shoulders of giants
+
+### Moral
+
+* Free as in speech, not as in beer
```
Comment (parkr) [inline _posts/2015-11-17-why-open-source.md]: obligation to give back? we learn from open source, so contributing is akin to teaching

File: _posts/2015-11-17-why-open-source.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+### Transparency
+
+* Gov: As relies increasingly on tech for citz. services, eseential to check work
+* Private: Voting machines, self driving cars
+
+### Participatory
+
+* Opportunity to shape policy (white house, congress, microsoft, etc.)
```
Comment (parkr) [inline _posts/2015-11-17-why-open-source.md]: reduce feelings of helplessness which cause apathy in many

**Comment by bkeepers** [comment]
Is this about consuming, contributing or both? 
- #thoughtleadership
- efficiency (working code you don't have to write)
- stability (more eyes, shallow bugs)

**Pull request merged**